frontend/src/modules/lf/segments/store/state.ts

```diff
@@ -1,30 +1,32 @@
 import { ProjectGroup, Project } from '@/modules/lf/segments/types/Segments';
 
 export interface SegmentsState {
-  selectedProjectGroup: ProjectGroup | null;
+  selectedProjectGroup: ProjectGroup | null
   adminProjectGroups: {
-    list: ProjectGroups[],
+    list: ProjectGroups[]
```

Comment:
I don't think this type exists, might be a typo

Reply:
yes it was a typo